source-dropbox/oauth2.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"strings"
+	"time"
+
+	pf "github.com/estuary/flow/go/protocols/flow"
+	"golang.org/x/oauth2"
+)
+
+func GetScopes() []string {
```

Comment:
There's no reason to have this be a function, and certainly not an exported one. You can just have it be a global variable: `var scopes = []string{"files.metadata.read", "files.content.read"}`